config/ext.json

```diff
@@ -153,7 +153,7 @@
     },
     "ldap": {
         "type": "builtin",
-        "arg-type": "with",
+        "arg-type": "with-prefix",
```

Comment:
this must be "with-ldap=/usr"

Reply:
Should be a `custom` here + add a new method `getUnixConfigureArg()` on the extension class.

Reply:
thanks, changed it